EvidenceApi.Tests/V1/E2ETests/DocumentSubmissionsTest.cs

```diff
@@ -246,5 +265,89 @@ public async Task ReturnBadRequestWhenSearchQueryIsInvalid()
+                               $"\"residentId\":\"{created.ResidentId}\"," +
+                               $"\"claimValidUntil\":{formattedValidUntil}," +
+                               $"\"retentionExpiresAt\":{formattedRetentionExpiresAt}," +
+                               $"\"state\":\"APPROVED\"," +
+                               "\"staffSelectedDocumentType\":{\"id\":\"passport-scan\",\"title\":\"Passport Scan\",\"description\":\"A valid passport open at the photo page\",\"enabled\":true}," +
+                               $"\"uploadPolicy\":{JsonConvert.SerializeObject(_createdUploadPolicy, Formatting.None)}," +
+                               $"\"document\":" + "{" + $"\"id\":\"{_document.Id}\",\"fileSize\":{_document.FileSize},\"fileType\":\"{_document.FileType}\"" + "}" +
+                               "}";
+
+            json.Should().Be(expected);
+        }
+
+        [Test]
+        public async Task CannotCreateDocumentSubmissionWithoutEvidenceRequestWithInvalidParameters()
+        {
+            var resident = TestDataHelper.Resident();
+            resident.Id = Guid.NewGuid();
+            DatabaseContext.Residents.Add(resident);
+            DatabaseContext.SaveChanges();
+
+            string body = "{" +
+                          $"\"residentId\": \"{resident.Id}\"," +
+                          "\"team\": \"\"," +
+                          "\"userCreatedBy\": \"test-user\"," +
+                          "\"staffSelectedDocumentTypeId\": \"passport-scan\"," +
+                          "\"documentName\": \"some document name\"," +
+                          "\"documentDescription\": \"some document description\"" +
+                          "}";
+            var jsonString = new StringContent(body, Encoding.UTF8, "application/json");
+            var uri = new Uri($"api/v1/document_submissions", UriKind.Relative);
+            var response = await Client.PostAsync(uri, jsonString);
+            response.StatusCode.Should().Be(400);
+        }
+
+        [Test]
+        public async Task CannotCreateDocumentSubmissionWithoutEvidenceRequestIfResidentDoesNotExist()
+        {
+            string body = "{" +
+                          $"\"residentId\": \"{Guid.NewGuid()}\"," +
+                          "\"team\": \"Development Housing Team\"," +
+                          "\"userCreatedBy\": \"test-user\"," +
+                          "\"staffSelectedDocumentTypeId\": \"passport-scan\"," +
+                          "\"documentName\": \"some document name\"," +
+                          "\"documentDescription\": \"some document description\"" +
+                          "}";
+            var jsonString = new StringContent(body, Encoding.UTF8, "application/json");
+            var uri = new Uri($"api/v1/document_submissions", UriKind.Relative);
+            var response = await Client.PostAsync(uri, jsonString);
+            response.StatusCode.Should().Be(400);
```

Comment:
might be worth  also checking the response content to make it clear that it's a different result from CannotCreateDocumentSubmissionWithoutEvidenceRequestWithInvalidParameters, since it's the same error code?
See here https://github.com/LBHackney-IT/evidence-api/blob/0b8e2bf2c2ed838b38d570b976cbdf1c569cf00f/EvidenceApi.Tests/V1/E2ETests/DocumentTypesTests.cs#L40 for an example.

Reply:
Addressed